CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md).
 
 ## [Unreleased]
+### Fixed
+- Remove unsupported Ruby version 2.1
+- Match sensu supported versions https://github.com/sensu/sensu/blob/master/.travis.yml#L4-L7
```

Comment:
we have our own policy we follow which is non EOL versions so this can be removed: https://docs.sensu.io/plugins/1.0/faq/#what-is-the-policy-on-supporting-end-of-life-eol-ruby-versions